### PR TITLE
BRIDGE-1165 Change wording in Tremor module

### PR DESCRIPTION
**Pull request by Erin-Mounts**
Change title from Tremor Test to Tremor Activity; show actual number of tasks user will be asked to perform

### CODE REVIEW

File: ResearchKit/Common/ORKOrderedTask.m [hunk 1]
```diff
@@ -59,6 +59,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import "ORKAccelerometerRecorder.h"
 #import "ORKAudioRecorder.h"
 #import "ORKWaitStep.h"
+#import <limits.h>
```
Comment (syoung-smallwisdom) [inline ResearchKit/Common/ORKOrderedTask.m:62]: ?

File: ResearchKit/Localized/en.lproj/ResearchKit.strings [hunk 2]
```diff
@@ -340,10 +340,15 @@
 "HOLE_PEG_TEST_TEXT_2" = "Lift fingers to drop the peg.";
 
 /* Tremor test active task. */
-"TREMOR_TEST_TITLE" = "Tremor Test";
+"TREMOR_TEST_TITLE" = "Tremor Activity";
```
Comment (syoung-smallwisdom) [inline ResearchKit/Localized/en.lproj/ResearchKit.strings:343]: Thank you. (smile)

File: ResearchKit/Localized/en.lproj/ResearchKit.strings [hunk 3]
```diff
@@ -340,10 +340,15 @@
...
 "TREMOR_TEST_INTENDED_USE" = "This activity measures the tremor of your hands in various positions. Perform this test with your more affected hand.";
 "TREMOR_TEST_INTRO_2_TEXT" = "Hold the phone as shown in the image below and find a place where you can sit comfortably for the duration of this task.";
-"TREMOR_TEST_INTRO_2_DETAIL" = "You will be asked to perform four tasks while in this position.";
+"TREMOR_TEST_INTRO_2_DETAIL_%" = "You will be asked to perform %@ while in this position.";
```
Comment (syoung-smallwisdom) [inline ResearchKit/Localized/en.lproj/ResearchKit.strings]: Might be worth some back-and-forth with Christine on the wording?

"You will be asked to perform %@ while sitting with the phone in your hand."

**Comment by syoung-smallwisdom** [comment]
Looks good.

One concern on the wording (that was already there).

File: ResearchKit/Localized/en.lproj/ResearchKit.strings (hunk 3 above)
Comment (Erin-Mounts) [inline ResearchKit/Localized/en.lproj/ResearchKit.strings]: Good point. I'll check with Christine.

**Comment by Erin-Mounts** [comment]
Christine agreed with your suggested wording so I updated to use that.

**Pull request merged**